tests/property/ic_api/cycles_burn/test/tests.ts

```diff
@@ -0,0 +1,45 @@
+import { getCanisterId } from 'azle/dfx';
+import { defaultPropTestParams, expect, it, Test } from 'azle/test';
+import fc from 'fast-check';
+
+import { execSyncPretty } from '../../../../../src/build/stable/utils/exec_sync_pretty';
```

Comment:
I think that since this is in a test it should be azle/whatever/exec_sync_prety instead of a relative path